.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -1,14 +1,12 @@
 ---
-name: Feature request
-about: Suggest an idea for this project
-title: ''
-labels: a:feature
```

Comment:
https://help.github.com/en/articles/manually-creating-a-single-issue-template-for-your-repository#adding-an-issue-template

> Note: You must put certain values in quotes if they are YAML reserved characters. For example, to add a label with an emoji in the label name such as `:bug:` Bug, you must add quotes "`:bug:` Bug" since `:` is a reserved character.